sourcegraph/sourcegraph
# feat(plg): remove accept invite page

Removes "/cody/invites/accept" page.

Invite processing is handled on the "/cody/manage" page and "/cody/invites/accept" is used only to redirect to the former.

Backend counterpart: https://github.com/sourcegraph/self-serve-cody/pull/886




## Test plan
- Tested manually
  - checkout branch fromhttps://github.com/sourcegraph/self-serve-cody/pull/886 and run SSC backend locally
  - run Sourcegraoh instance in dotcom mode locally
  - log in as a Cody Pro team admin and send an invite to another user
  - log in as an invited user and follow the link from the email
  - ensure link navigates to the "/cody/manage/" page and is hanled as expected (see test plan from https://github.com/sourcegraph/sourcegraph/pull/63231)  



## Changelog
